base/libgit2/walker.jl

```diff
@@ -1,5 +1,31 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+    GitRevWalker(repo::GitRepo)
+
+A `GitRevWalker` *walks* through the *revisions* (i.e. commits) of
+a git repository `repo`. It is a collection of the commits
+in the repository, and supports iteration and calls to [`map`](@ref)
+and [`count`](@ref) (for instance, `count` could be used to determine
+what percentage of commits in a repository were made by a certain
+author).
+
+# Examples
+```julia
+oids = LibGit2.with(LibGit2.GitRevWalker(repo)) do walker
+    LibGit2.map((oid,repo)->string(oid), walker, by = LibGit2.Consts.SORT_TIME)
+end
+```
+Here, `map` visits each commit using the `GitRevWalker` and finds its [`GitHash`](@ref).
+
+```julia
+cnt = LibGit2.with(LibGit2.GitRevWalker(repo)) do walker
+    count((oid,repo)->(oid == commit_oid1), walker, oid=commit_oid1, by=LibGit2.Consts.SORT_TIME)
```

Comment:
and be module-qualified I guess if the example is intended to be runnable from outside the LibGit2 module (I hope `LibGit2.count` isn't extending `Base.count` with libgit2-specific kwargs?)